### PR TITLE
extracted max gas limit for vm query to config

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
extracted the maximum gas limit to be used for vm queries inside config.toml

### CODE REVIEW

**Review by AdoAdoAdo — APPROVED** [review]

**Review by sasurobert — DISMISSED** [review]

File: process/smartContract/scQueryService_test.go [hunk 1]
```diff
@@ -308,6 +308,85 @@ func TestExecuteQuery_ReturnsCorrectly(t *testing.T) {
...
+			FuncName:  "function",
+			Arguments: [][]byte{},
+		}
+
+		_, err := target.ExecuteQuery(&query)
+		require.Nil(t, err)
+		require.True(t, runSCWasCalled)
+	})
+
+	t.Run("custom gas defined, should use max uint64", func(t *testing.T) {
```
Comment (AdoAdoAdo) [inline process/smartContract/scQueryService_test.go:350]: should not use

**Review by AdoAdoAdo — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed .
@@                            Log scanner                                  @@
##                            max-gas-limit-vmquery-config-value                                       ##
================================================================================
- Known Warnings              10
- New Warnings                2
- Known Errors                0
- New Errors                  0
- Panics                      0
================================================================================

**Review by sasurobert — DISMISSED** [review]

File: cmd/node/config/config.toml [hunk 2]
```diff
@@ -588,6 +588,9 @@
         SameSourceRequests = 10000
         # SameSourceResetIntervalInSec time frame between counter reset, in seconds
         SameSourceResetIntervalInSec = 1
+        # MaxGasPerVmQuery defines the maximum amount of gas to be allocated for VM Queries comming from API
+        # If set to 0, then MaxUInt64 will be used
+        MaxGasPerVmQuery = 1500000000  #1.5b
```
Comment (iulianpascalau) [inline cmd/node/config/config.toml:593]: This has nothing to do with the antiflood param. Please move it in the vm queries configs from api

**Review by iulianpascalau — CHANGES_REQUESTED** [review]

File: cmd/node/config/config.toml (hunk 2 above)
Comment (bogdan-rosianu) [reply cmd/node/config/config.toml]: moved

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed. 
Warnings are present before the upgrade. 

@@                            Log scanner                                  @@
##                            max-gas-limit-vmquery-config-value           ##
================================================================================
- Known Warnings              17
- New Warnings                3
- Known Errors                0
- New Errors                  0
- Panics                      0
================================================================================

**Review by AdoAdoAdo — APPROVED** [review]

**Pull request merged**